### ISSUE TITLE
Source Language: is empty - is this regular behavior  

### ISSUE THREAD

**Issue by shivaylamba** [dylibso/modsurfer#31]
<img width="1197" alt="image" src="https://user-images.githubusercontent.com/19529592/227468401-9e123b52-a768-41b4-b89c-f6b4ce262492.png">
I uploaded a Rust compiled WASMEdge webassembly module but the source language is unknown, is this regular behavior for now?

**Comment by nilslice** [comment]
Yes, sometimes the binary is stripped of particular "sections", one of which is where our language detection looks. If this section is not available, or we cannot detect the language, then it will display as "Unknown".
